src/OpenTracing/Mock/MockSpan.cs

```diff
@@ -157,6 +158,30 @@ public ISpan SetTag(string key, string value)
             return SetObjectTag(key, value);
         }
 
+        public ISpan SetTag(BooleanTag tag, bool value)
+        {
+            tag.Set(this, value);
```

Comment:
Should we just call `SetObjectTag(tag.Key, value)` in these new methods? Seems more readable than doing the additional jump through `tag.Set()`.

Reply:
@Falco20019 what do you think? Could you update this if you agree?

Reply:
Sorry for the delay. Will do it tomorrow :) was out of office the last days and had some other projects to work on yesterday and today.